lib/logic.rb

```diff
@@ -0,0 +1,149 @@
+class Board
+  attr_accessor :board
+
+  def initialize
+    @turn = true
+    @board = [
+      [' ', ' ', ' '],
+      [' ', ' ', ' '],
+      [' ', ' ', ' ']
+    ]
+  end
+
+  def print_board
```

Comment:
We don't wanna mix our display related code with our classes. Classes should just be responsible for logic.